data/json/recipes/recipe_deconstruction.json

```diff
@@ -2257,5 +2257,138 @@
     "time": 500,
     "components": [ [ [ "paper", 10 ] ] ],
     "flags": [ "BLIND_EASY" ]
+  },
+  {
+    "result": "spectrophotometer",
+    "type": "uncraft",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "skills_required": [ "cooking", 1 ],
```

Comment:
I assume you meant that this recipe requires chemistry? I think chemistry in form of cooking begins at level 3.